recipes-containers/docker/docker.inc

```diff
@@ -77,23 +78,26 @@ do_compile() {
 
 	export DISABLE_WARN_OUTSIDE_CONTAINER=1
 
-	cd ${S}/src/import/
-
 	# this is the unsupported built structure
 	# that doesn't rely on an existing docker
 	# to build this:
 	VERSION="${DOCKER_VERSION}" DOCKER_GITCOMMIT="${DOCKER_COMMIT}" ./hack/make.sh dynbinary
 
-        # build the cli
+	# build the cli
 	cd ${S}/src/import/.gopath/src/github.com/docker/cli
 	export CFLAGS=""
 	export LDFLAGS=""
 	export DOCKER_VERSION=${DOCKER_VERSION}
-	VERSION="${DOCKER_VERSION}" DOCKER_GITCOMMIT="${DOCKER_COMMIT}" make dynbinary
+	VERSION="${DOCKER_VERSION}" DOCKER_GITCOMMIT="${DOCKER_COMMIT}" make binary
```

Comment:
I don't understand what this change from "make dynbinary" to "make binary" does, but I'm surprised to see it here since it doesn't seem related to the buildx-plugin change. Is this change intentional?

Reply:
reverted back to dynbinary